### DISCUSSION TITLE
Change notification sound

### DISCUSSION THREAD

**Discussion by flaviozluca** [Intermesh/groupoffice#1032] [Q&A]
Friends,

How is everyone? Is there a way to change the notification sound (calendar alerts)?

Thanks.

**Comment by Gnggi** [comment]
As long as there is no option to change it in GO you could probably change it in filesystem located in GO directory  and subdirectory 'views/Extjs3/themes/Paper/sounds/'. The sounds are in both .ogg and .mp3 type.

**Comment by flaviozluca** [comment]
Thanks. It would be nice to have a way via the interface, for users that dont have access to the system.
